scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -191,9 +171,6 @@
     become: true
     become_user: ardana
 
-  #- name: Output of config-processor-run.yml playbook
-  #  debug: var
-
```

Comment:
(nit) This is unrelated to the subject of the commit message

Reply:
splitted into a separate commit